Yosemite/Yosemite/Stores/ProductStore.swift

```diff
@@ -495,21 +495,21 @@ private extension ProductStore {
         upsertStoredProductsInBackground(readOnlyProducts: [product], siteID: product.siteID, onCompletion: onCompletion)
     }
 
-    /// Checks if the store is eligible for products onboarding.
+    /// Checks if the store already has any products.
     /// Returns `true` if the store has no products.
```

Comment:
I think we have to update this comment.

Reply:
Nice catch! Updated in 339bae8c8a10318bd06041ed06baed8dc30129ae.